src/pf-notification-drawer/pf-notification-drawer.scss

```diff
@@ -0,0 +1,46 @@
+.drawer-pf {
```

Comment:
This is a question regarding the overall architecture of patternfly-webcomponents. Why are we using SCCS instead of LESS? 

I don't believe we've added anything specific to SCCS? We should be consuming Patternfly LESS so we can take advantage of color variables for example. That is, if we must create custom CSS that's not included in Patternfly?